lib/fluent/plugin/in_s3.rb

```diff
@@ -76,8 +76,8 @@ def initialize
       desc "Profile name. Default to 'default' or ENV['AWS_PROFILE']"
       config_param :profile_name, :string, default: nil
     end
-    desc "S3 bucket name"
-    config_param :s3_bucket, :string
+    desc "S3 bucket name(s) separated by commas in case of multiple bucket names"
+    config_param :s3_buckets, :string
```

Comment:
Changing parameter name breaks existing configurations.
We couldn't accept this change.
Instead, using deprecated attr in config_param and adding parameter conversion code are needed to accept this PR.

Reply:
Thank you for the review. I'll do the requested changes to make it backwards compatible and get back to you 👍